src/ibc/lightclients/tendermint/v1/tendermint.ts

```diff
@@ -375,75 +434,88 @@ export const ConsensusState = {
 };
 
 function createBaseMisbehaviour(): Misbehaviour {
-  return { clientId: "", header1: undefined, header2: undefined };
+  return {
+    clientId: "",
+    header_1: undefined,
+    header_2: undefined,
```

Comment:
the `header1` => `header_1` may only matter for amino encoders. Not for the proto.